CoreServices/ASErrorHandler.rpy

```diff
@@ -4,40 +4,77 @@
 # Copyright: (C) 2018
 # RSOD-related files
 init -1000 python:
-    aliceos_default_errors = {
-        "boot_fault": "INACCESSIBLE_BOOT_DEVICE",
-        "cyanide": "MISSING_CYANIDE_INSTRUMENT",
-        "label_fault": "LABEL_FAULT_IN_NONLABEL_AREA",
-        "setup_fail":"SETUP_ASSISTANT_FAIL"
-    }
+    class ASErrorTypes():
 
+        headers = {
+            "FS": "FILESYSTEM_",
+            "Screen": "SCREEN_",
+            "SIP": "SEC_ADD_",
+            "Setup": "SETUP_"
+        }
+
+        fserror = {
+            "NoPerm": "INSUFFICIENT_PERMISSIONS",
+            "UnreadableSector": "SECTOR_INVALID",
+            "CriticalFileMissing": "CRITICAL_FILE_MISSING"
+        }
+
+        screrr = {
+           "UnterminatedProgress": "PROGRESS_VOID",
+           "MissingScreen": "SCREEN_UNDEFINED",
+           "ScreenCorrupt": "SCREEN_INVALID" 
+        }
+
+        siperr = {
+            "SIPComponentFailure": "SIP_FAILED_INIT",
+            "SIPCriticalFileCheckFail": "SIP_CRITICAL_FILE_INVALID",
+            "SIPComponentProfileFail": "SIP_COMPONENT_FAILED"
+        }
+
+        pisaerr = {
+            "NullInput": "INPUT_REUTRNED_NULL"
```

Comment:
Change this to `NULL_IN_NON_NULLABLE_CTX`

Reply:
That is human-readable, but I don't think it's quite human-_understandable_...

Reply:
It's for programmer purposes so I don't think a end-user would need to learn about that.